src/bluesky/tests/test_seq_num_indices_in_sync.py

```diff
@@ -43,7 +43,7 @@ def __init__(
         self.compose_stream_resource = ComposeStreamResource()
         if DRAFT_0_STREAM_RESOURCE:
             self.stream_resource_compose_datum_pairs = tuple(
-                self.compose_stream_resource("", "", f"non_existent_{det}.hdf5", det, {}) for det in self.detectors
+                self.compose_stream_resource("", f"non_existent_{det}.hdf5", det, {}) for det in self.detectors
```

Comment:
I think this has to be how it was before since if `DRAFT_0_STREAM_RESOURCE`, `event-model < 1.21.0` and the method takes different args:

https://github.com/bluesky/event-model/blob/cb6896fb5033e0ac64fdddde48fa805d3ab91c90/event_model/__init__.py#L2087-L2091

vs

https://github.com/bluesky/event-model/blob/145118722a2e2f15abf32010e3e6b71506398140/event_model/__init__.py#L2083-L2088

Add an ignore comment instead.

Reply:
Ah I see that now, thanks!